src/main/java/dev/emi/trinkets/mixin/CreativeInventoryScreenMixin.java

```diff
@@ -51,7 +52,7 @@ private int size(DefaultedList<ItemStack> list) {
 
 	@Inject(at = @At("HEAD"), method = "setSelectedTab")
 	private void setSelectedTab(ItemGroup g, CallbackInfo info) {
-		if (g != ItemGroups.INVENTORY) {
+		if (g.getType() == ItemGroup.Type.INVENTORY) {
```

Comment:
inverted?

Reply:
Not sure how I didn't notice this, will fix.